torch/testing/_internal/common_utils.py

```diff
@@ -1528,6 +1528,14 @@ def random_symmetric_pd_matrix(matrix_size, *batch_dims, **kwargs):
         + torch.eye(matrix_size, dtype=dtype, device=device) * 1e-5
 
 
+def random_hermitian_pd_matrix(matrix_size, *batch_dims, **kwargs):
```

Comment:
Add a comment describing what this function does and how to call it.

Reply:
I think the signature of this function should be: `(matrix_size, *, batch_dims, dtype=None, device='cpu')`

Reply:
Okay, I can change that. I was just following the other existing functions.
```python
def random_symmetric_pd_matrix(matrix_size, *batch_dims, **kwargs):
    dtype = kwargs.get('dtype', torch.double)
    device = kwargs.get('device', 'cpu')
    A = torch.randn(*(batch_dims + (matrix_size, matrix_size)),
                    dtype=dtype, device=device)
    return torch.matmul(A, A.transpose(-2, -1))
```
In fact, all calls to `random_symmetric_pd_matrix` can be safely replaced by `random_hermitian_pd_matrix`, probably in a follow-up PR.